libvast_test/vast/test/test.hpp

```diff
@@ -116,31 +133,24 @@ struct less_equal_compare {
 // Checks that continue with the current test on failure
 #define CHECK CAF_CHECK
 #define CHECK_EQUAL(x, y)                                                      \
-  CAF_CHECK_FUNC(vast::test::detail::equality_compare, (x), (y))
-#define CHECK_NOT_EQUAL(x, y)                                                  \
-  CAF_CHECK_FUNC(vast::test::detail::inequality_compare, (x), (y))
-#define CHECK_LESS(x, y)                                                       \
-  CAF_CHECK_FUNC(vast::test::detail::less_compare, (x), (y))
-#define CHECK_LESS_EQUAL(x, y)                                                 \
-  CAF_CHECK_FUNC(vast::test::detail::less_equal_compare, (x), (y))
-#define CHECK_GREATER(x, y)                                                    \
-  CAF_CHECK_FUNC(vast::test::detail::greater_compare, (x), (y))
-#define CHECK_GREATER_EQUAL(x, y)                                              \
-  CAF_CHECK_FUNC(vast::test::detail::greater_equal_compare, (x), (y))
-#define CHECK_NOERROR(x)                                                       \
   do {                                                                         \
-    if (!(x)) {                                                                \
-      CAF_CHECK_FAILED((x).error());                                           \
-    } else {                                                                   \
-      CAF_CHECK_PASSED(#x);                                                    \
-    }                                                                          \
+    using namespace vast;                                                      \
+    CAF_CHECK_EQUAL((x), (y));                                                 \
   } while (false)
+#define CHECK_NOT_EQUAL(x, y) CAF_CHECK_NOT_EQUAL((x), (y))
+#define CHECK_LESS(x, y) CAF_CHECK_LESS((x), (y))
+#define CHECK_LESS_EQUAL(x, y) CAF_CHECK_LESS_OR_EQUAL((x), (y))
+#define CHECK_GREATER(x, y) CAF_CHECK_GREATER((x), (y))
+#define CHECK_GREATER_EQUAL(x, y) CAF_CHECK_GREATER_OR_EQUAL((x), (y))
 #define CHECK_ERROR(x) CHECK_EQUAL(!(x), true)
 #define CHECK_SUCCESS(x) CHECK_EQUAL((x), caf::none)
 #define CHECK_FAILURE(x) CHECK_NOT_EQUAL((x), caf::none)
-#define CHECK_FAIL CAF_CHECK_FAIL
```

Comment:
CAF_CHECK_FAIL was removed and the 0.17.6 implementation is quite weird. Didn't seem to be worth using anyways

Reply:
I don't think I've ever used this, so that makes sense.

Reply:
I don't think I've ever used this, so that makes sense.